group-generators/helpers/data-providers/index.ts

```diff
@@ -34,6 +35,7 @@ import wiwBadgeInterfaceSchema from "./wiw-badge/interface-schema.json";
 import { DataProviders } from "topics/data-provider";
 
 export const dataProviders = {
+  AttestationStationProvider,
   BigQueryProvider,
   EnsProvider,
   EthLeaderboardProvider,
```

Comment:
And you have also too instantiate the function + countfunction of your data provider in `dataProvidersAPIEndpoints`